pybaseball/team_fielding.py

```diff
@@ -30,9 +30,7 @@ def team_fielding(start_season, end_season=None, league='all', ind=1):
 
     fg_data = fangraphs.get_fangraphs_tabular_data_from_url(
         _FG_TEAM_FIELDING_URL.format(start_season=start_season, end_season=end_season, league=league, ind=ind),
-        '//table[@class="rgMasterTable"]',
-        percentage_columns=['CS%', 'lgCS%'],
-        non_numeric_columns=['Team', 'Name']
+        '//table[@class="rgMasterTable"]'
```

Comment:
Substantive: is this always "rgMasterTable"? Can we streamline this call?

Reply:
Great question. Let me check.

Reply:
So here is all the places we'd end up implementing this in the future:

- batting_leaders.py
- pitching_leaders.py
- team_pitching.py

All of them use this table class. So I could have this as a default param. I'll modify it shortly.